docs/docsite/rst/roadmap/COLLECTIONS_4.rst

```diff
@@ -0,0 +1,49 @@
+===================
+Ansible project 4.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_11` for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2021-01-26: New Collections will be reviewed for inclusion in ansible-4.
+:2021-03-03: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
+:2021-03-17: ansible-4.0.0 alpha2
```

Comment:
were these two dates intentionally on Wednesday or should they be shifted to Tuesday?

```suggestion
:2021-03-02: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
:2021-03-16: ansible-4.0.0 alpha2
```

Reply:
Yes... It's easier to see when I have the ansible-core pre-releases interspersed: https://hackmd.io/y7BBcweNR3aRVLuMbKkDxw#Proposed-Schedule-for-Ansible-40   The idea is for the ansible alphas to release the day after the ansible-core betas and release candidcates.

Reply:
Okay, relrod has finalized the dates for alpha2 and so forth: https://github.com/ansible/ansible/pull/73560/files

ansible-core-2.11-beta1 will still be on a Tuesday but subsequent releases will be shifted to Monday.  So I'll change out ansible-4.0.0 alpha2 to be a Tuesday.

About why beta1 is on a Tuesday, it is a combination of publishing that date quite a few months ago so it seems too late to change and it gives people one extra day to get their work finished before the beta freeze.